src/Student.cpp

```diff
@@ -0,0 +1,64 @@
+#include "Student.hpp"
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+Student::Student(){
+    name = "x";
+    nick = "y";
+    gitHub = "none";
+    firecode = "none";
+}
```

Comment:
We don't want such students :) This is invalid data, so it's better not to have a default constructor without parameters.